src/components/Subscribe/index.tsx

```diff
@@ -0,0 +1,58 @@
+import clsx from "clsx";
+import React, { useState } from "react";
+
+import styles from "./styles.module.css";
+
+const submitFormSubscription = (
```

Comment:
why not inside component?

Reply:
here could be some more complex logic, so I've separated this function